PWGUD/Core/UPCJpsiCentralBarrelCorrHelper.h

```diff
@@ -0,0 +1,215 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \brief
+/// \author Sara Haidlova, sara.haidlova@cern.ch
+/// \since March 2024
+
+#ifndef PWGUD_CORE_UPCJPSICENTRALBARRELCORRHELPER_H_
+#define PWGUD_CORE_UPCJPSICENTRALBARRELCORRHELPER_H_
+
+#include <algorithm>
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+using namespace std;
```

Comment:
Never put `using namespace` in headers. It pollutes the namespace in every file that includes it.